Source/JavaScriptCore/llint/LowLevelInterpreter.asm

```diff
@@ -1694,6 +1694,76 @@ else
 end
     doVMEntry(makeJavaScriptCall)
 
+if (ARM64E or ARM64) and ADDRESS64
+    macro vmEntryToJavaScriptSetup()
+        functionPrologue()
+        pushCalleeSaves()
+        vmEntryRecord(cfr, sp)
+        storep a1, VMEntryRecord::m_vm[sp]
+        loadpairq VM::topCallFrame[a1], t8, t9 # topCallFrame and topEntryFrame
+        storepairq t8, t9, VMEntryRecord::m_prevTopCallFrame[sp]
+    end
+
+    # EncodedJSValue vmEntryToJavaScriptWith0Arguments(void*, VM*, CodeBlock*, JSObject*, JSValue, JSValue)
+    # entry, vm, codeBlock, callee, thisValue, arg0, arg1
```

Comment:
This comment is wrong. it's 0 args, so I think it should be:
```
    # EncodedJSValue vmEntryToJavaScriptWith0Arguments(void*, VM*, CodeBlock*, JSObject*, JSValue)
    # entry, vm, codeBlock, callee, thisValue
```

Reply:
Fixed.